cumulusci/tasks/salesforce/InstallPackageVersion.py

```diff
@@ -31,6 +31,9 @@ class InstallPackageVersion(Deploy):
         "retry_interval_add": {
             "description": "Number of seconds to add before each retry (default=30),"
         },
+        "security_type": {
+            "description": "Which users to install package for (FULL = all users, NONE = admins only)"
```

Comment:
Let's expose `PUSH` too so we can replicate the user push upgrade experience in QA.